connexion/decorators/response.py

```diff
@@ -52,9 +52,10 @@ def validate_response(self, data, status_code, headers, url):
 
         if response_definition and response_definition.get("headers"):
             # converting to set is needed to support python 2.7
-            response_definition_header_keys = set(response_definition.get("headers").keys())
+            required_header_keys = {k for (k, v) in response_definition.get("headers").items()
+                                    if 'required' not in v or v['required']}
```

Comment:
you can simplify this by using `dict.get()`:
```
if v.get('required')
# or to be explicit:
if v.get('required', False)
```

Reply:
Great tip, changed